### ISSUE TITLE
Compilation with ITK 5?

### ISSUE THREAD

**Issue by mabelzunce** [SuperElastix/SimpleElastix#337]
Hello
I was wondering if SimpleElastix is compatible with ITK 5?
I've been unsuccessfully trying to compile SimpleElastix using ITK 5. The main reason I'm trying to do that because there is a bug in SimpleITK that is solved with ITK 5:
https://github.com/SimpleITK/SimpleITK/issues/750

Thanks
Martin

**Comment by MarHHM** [comment]
> Hello
> I was wondering if SimpleElastix is compatible with ITK 5?
> I've been unsuccessfully trying to compile SimpleElastix using ITK 5. The main reason I'm trying to do that because there is a bug in SimpleITK that is solved with ITK 5:
> [SimpleITK/SimpleITK#750](https://github.com/SimpleITK/SimpleITK/issues/750)
> 
> Thanks
> Martin

i've tried the same (VS on Win 10) & SimpleITK didn't build (was giving me strange errors about .h files (e.g. itkImageSource.h - itkImage.h ... etc..) not found, although they were there & the paths to them were correct).

What i discovered when i built it again through the "SuperBuild configuration" is that the elastix version fetched using this method is 4.9.. & that's how i concluded that SimpleITK is not yet ready for elastix 5.0 (& ITK 5.0)..

@kaspermarstal   would u please confirm this ??  :)

**Comment by kaspermarstal** [comment]
Yes that is correct, the new version of SimpleITK and elastix (which does support ITKv5 now) has not been merged into this repo. Contributions are welcome! (otherwise I will do it when I get around to it but I don't have much time maintenance these days)
